quill/quill.d.ts

```diff
@@ -1,3 +1,8 @@
+// Type definitions for quill.js
+// Project: https://github.com/quilljs/quill/tree/0.20.1
+// Definitions by: Mike Morearty <https://github.com/mmorearty>
```

Comment:
Do you know the set of circumstances by which you are git blamed for all the code and how I might find the actual author?